test/index.test.js

```diff
@@ -3,120 +3,174 @@
 const expect = require('chai').expect;
 const rewire = require('rewire');
 
-const logs = [];
+const PrettyStream = require('bunyan-prettystream');
 
 describe('index.js', () => {
   let oldStdoutWrite;
-  let logger;
-
-  before(() => {
-    oldStdoutWrite = process.stdout.write;
-
-    process.stdout.write = function stubStdout(string) {
-      try {
-        string = JSON.parse(string);
-        logs.push(string);
-      } catch (e) {
-        oldStdoutWrite.apply(process.stdout, arguments);
-      }
-    };
-
-    const oldEnv = process.env;
-    process.env = {
-      LOGGER_NAME: 'Test logger name',
-      LOGGER_LEVEL: 'debug',
-      LOGENTRIES_TOKEN: '00000000-0000-0000-0000-000000000000',
-      SENTRY_DSN: 'https://a:b@fake.com/12345'
-    };
-    logger = rewire('../index');
-    process.env = oldEnv;
-  });
 
-  after(() => {
-    process.stdout.write = oldStdoutWrite;
-  });
+  describe('Log functions', () => {
+    let logger;
+    const logs = [];
+
+    before(() => {
+      oldStdoutWrite = process.stdout.write;
+
+      process.stdout.write = function stubStdout(string) {
```

Comment:
The problem here is that if we stub `process.stdout.write`, nothing will be output to the console, including mocha